src/components/button/index.ts

```diff
@@ -1,3 +1,18 @@
 const Button = () => {};
 
 export default Button;
+
+/*
+ * @param buttons - array of card-buttons
+ * this adds an alert functionality that contains the card headline on click of a button
+ */
+export function headlineOnClick(buttons) {
```

Comment:
You need the whole card in this function. So I would give the whole cardlist as payload